README.md

```diff
@@ -1,6 +1,168 @@
-# cemerson/auth
+# Auth
 
-A framework & datastorage independent authentication library for PHP
+A framework & datastorage independent authentication library for PHP.
+
+## Authentication vs Authorisation
+
+This package performs only authentication - ensuring your application users are who they say they are.
+
+You will need to handle authorisation - what your users can and can't do when logged in - elsewhere in your application.
+
+## Installation
+
+Installation is via [Composer](https://getcomposer.org/):
+
+```bash
+$ composer require cemerson/auth
+```
+
+or add it by hand to your `composer.json` file.
```

Comment:
Can you remove this line - interaciton with composer should always be with the command line where possible, and hand-editing of composer.json should be discouraged.